homeassistant/components/weatherflow/sensor.py

```diff
@@ -80,11 +79,10 @@ def get_native_value(self, device: WeatherFlowDevice) -> datetime | StateType:
     WeatherFlowSensorEntityDescription(
         key="air_density",
         translation_key="air_density",
-        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-        device_class=SensorDeviceClass.VOLATILE_ORGANIC_COMPOUNDS,
+        native_unit_of_measurement="kg/m³",
         state_class=SensorStateClass.MEASUREMENT,
-        suggested_display_precision=3,
-        raw_data_conv_fn=lambda raw_data: raw_data.m * 1000000,
+        suggested_display_precision=5,
+        raw_data_conv_fn=lambda raw_data: raw_data.magnitude,
```

Comment:
I don't understand the reason for `.magnitude` here, but probably I am missing something.